src/components/Modal/EditOrder.tsx

```diff
@@ -77,15 +79,17 @@ class EditOrder extends React.Component<EditOrderProps, EditOrderState> {
       priceAccuracy: pathOr(2, ['accuracy'], currentInstrument),
       quantityAccuracy: pathOr(2, ['baseAsset', 'accuracy'], currentInstrument)
     };
-    this.baseName = currentInstrument.name.split('/')[0];
-    this.quoteName = currentInstrument.name.split('/')[1];
+    this.baseName = currentInstrument.baseAsset.name;
```

Comment:
could you plz rename to `base(quote)AssetId` and `base(quote)AssetName`? just to make it clearier